src/multi_vector_simulator/utils/constants.py

```diff
@@ -260,23 +260,39 @@
 
 ENERGY_CARRIER_UNIT = "energy_carrier_unit"
 DEFAULT_WEIGHTS_ENERGY_CARRIERS = {
-    "Electricity": {UNIT: "kWh_eleq/kWh_el", VALUE: 1, ENERGY_CARRIER_UNIT: "kWh_el"},
-    "Heat": {UNIT: "KWh_eleq/kWh_therm", VALUE: 1, ENERGY_CARRIER_UNIT: "kWh_therm"},
+    "LNG": {UNIT: "kWh_eleq/kg", VALUE: 33.4728198, ENERGY_CARRIER_UNIT: "kg",},
```

Comment:
It seems to me that this global variable is like a csv (with headers: carriers, value, unit, carrier_unit), which means we can  use the csv directly in `Model_Assumption.rst`.